.circleci/scripts/config-template.yml

```diff
@@ -143,6 +143,51 @@ jobs: # Each project will have individual jobs for each specific task it has to
                 paths:
                   - speckle-sharp-ci-tools/Installers
 
+  build-connector-new: # Reusable job for basic connectors
```

Comment:
Now that we have this working, let's try to come up with a more elegant solution than duplicating the entire connector build step.